src/include/msvc.h

```diff
@@ -110,6 +110,26 @@ __wt_atomic_storebool(bool *vp, bool v)
     *(vp) = (v);
 }
 
+/*
+ * __wt_atomic_loadvbool --
+ *     Atomically read a volatile boolean.
+ */
+static inline bool
+__wt_atomic_loadvbool(volatile bool *vp)
+{
+    return (*(vp));
```

Comment:
I like the distinct lack of atomics, maybe clarify in the comment?

Reply:
Clarified in [0f3eca7](https://github.com/wiredtiger/wiredtiger/pull/10328/commits/0f3eca7a93f962e6ac18095b8e1435ad1f7b4b8b)